scientific_library/tvb/tests/library/simulator/regionmapping_test.py

```diff
@@ -0,0 +1,56 @@
+"""
+Tests full region mapping when both surface and sub-cortical regions are present.
+
+.. moduleauthor:: Borana Dollomaja <borana.dollomaja@univ-amu.fr>
+"""
+import pytest
+import numpy
+import time
+# import TVB modules
+from tvb.simulator.lab import *
+from tvb.datatypes.region_mapping import RegionMapping
+from tvb.tests.library.base_testcase import BaseTestCase
+
+class SetUpSimulator(object):
+    """
+    Simulator test class for region mapping
+    """
+    def __init__(self, monitors=(monitors.Raw(),)):
```

Comment:
instead of placing this in a separate class, can you just make this the `setUp` method on the TestCase below?